package.json

```diff
@@ -8,8 +8,9 @@
     "test": "tests"
   },
   "scripts": {
-    "e2e": "playwright test",
-    "e2e:report": "playwright show-report",
+    "//pwtests": "The version of the docker playwright image used here must exactly match the versions of @playwright/test in devDependencies and the Github Actions configuration (in playwright.yml).  The version of playwright used in devDependencies should not include the ^ prefix, so it doesn't auto-upgrade and get out of sync.",
```

Comment:
Oh, I'd forgotten `package.json` was a format that doesn't have comments. No strong opinion between keeping this and removing it.

Reply:
I think the comment about consistent versions is very important.  We could stuff it in playwright.yml instead, but it is more likely to be found here.